magento/magento2
# Remove sku from operators to validate condition

### Description
@AndersonAlves faced an issue with our customers running Magento 2.2. When you create a product with a sku and you use the sku as a condition to a cart rule it will use the "{}"(contains) condition, and not equals "==". We debugged the code and we found an issue available only on Magento 2.2.

### Fixed Issues
This PR will fix this issue on Magento 2.2. Magento 2.1 and 2.3 are working correctly, this fix was already applied to them.

### Manual testing scenarios
1. Create a product with a sku 123
2. Create a configurable product with sku `conf123`.
3. Create simple products for this configurable with `123` inside the skus, for example `conf-red-123` and `conf-blue-123`.
4. Create a cart rule and inside actions add a condition to apply the discount only if a product sku is **equals** to `123`.
5. You can specify a coupon code or not for this rule, it's optional.
6. Specify a discount amount or a percentage.
7. Add the configurable product to the cart and go to the cart page.
8. You will see the discount is being applied to the configurable product, and the configurable product sku is not equals to `123`, it contains `123` in the sku, but is not equal.
